packages/vuetify/src/components/VList/VListChildren.tsx

```diff
@@ -49,9 +49,12 @@ export const VListChildren = genericComponent<new <T extends InternalListItem>()
         title: slots.title ? (slotProps: any) => slots.title?.({ ...slotProps, item }) : undefined,
       }
 
+      const [listGroupProps, _1] = filterListGroupProps(itemProps as any)
+
       return children ? (
         <VListGroup
           value={ itemProps?.value }
```

Comment:
Is this already included in listGroupProps?

Reply:
Yes but everything blows up when I remove it.